### PR TITLE
Added Particles.vue

### PR DESCRIPTION
**Pull request by matteobruni**
#### `Open Source`

- [x] Link description does not contain a link to an author / third-party resource
- [x] The documentation (README) contains a description of the project, illustration of the project with a demo or screenshots and a CONTRIBUTING section
- [x] The documentation is in English.
- [x] The project is active and maintained.
- [x] The project accepts contributions.
- [x] Not a commercial product

### CODE REVIEW

**Comment by matteobruni** [comment]
Is it possible to replace `vue-particles` with `Particles.vue`? It's almost the same library but the first one was abandoned about 3 years ago, the other is actively maintained

**Comment by Scrum** [comment]
@matteobruni No, because authorship belongs to another user.

Yes, we can remove the package because it hasn't been supported for a long time, but your package should still be added to the end of the list.

**Comment by matteobruni** [comment]
Fixed

File: README.md [hunk 1]
```diff
@@ -1742,7 +1742,8 @@ _Display non-editable events in a Calendar_
 - [vue-text-middle-ellipsis](https://github.com/Tardigrada777/vue-text-middle-ellipsis) - A simple vue-directive(2+) that clamps the content in the middle of the element if the content inside is too long.
 - [vuesence-book](https://github.com/altrusl/vuesence-book) - Documentation/help system component.
 - [vue-turnjs](https://github.com/Reidond/vue-turnjs) - A component libary to add book experience to your app.
-- [vuesence-sliding-header](https://github.com/altrusl/vuesence-sliding-header) - Sliding header(s) for top navigation bar(s).
+- [vuesence-sliding-header](https://github.com/altrusl/vuesence-sliding-header) - Sliding header(s) for top navigation bar(s
```
Comment (Scrum) [inline README.md:1745]: Please correct

Comment (matteobruni) [reply README.md]: Fixed, sorry about that!

**Pull request merged**